rero/rero-ils
# Implement stats for pricing

## Why are you opening this PR?

- https://tree.taiga.io/project/rero21-reroils/us/1848?milestone=294722

## How to test?

- logged in as admin@rero.ch
- go to the tools menu and select statistics
- explore the interface
- try to modify the data (loans, documents, etc.) go to the live view and see that the values are changed.

## Code review check list
